.github/workflows/godot-linux.yml

```diff
@@ -0,0 +1,61 @@
+name: "Godot Linux"
+on:
+  workflow_dispatch:
+  pull_request:
+  release:
+    types:
+      - released
+
+env:
+  GODOT_VERSION: 3.6
+  EXPORT_NAME: rota-linux
+
+jobs:
+  export:
+    name: Export
+    runs-on: ubuntu-latest
+    container:
+      image: docker.io/smks/godot-ci:3.6
```

Comment:
There's a [setup-godot github action](https://github.com/marketplace/actions/setup-godot-action) that seems to [work fine in our usage](https://github.com/endlessm/godot-block-coding/blob/main/.github/workflows/checks.yaml#L41).